src/test/java/uk/gov/ons/ctp/response/casesvc/message/CaseReceiptReceiverIT.java

```diff
@@ -41,9 +29,18 @@
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
 public class CaseReceiptReceiverIT {
 
-  private UUID collectionExerciseId;
+  private static PubSubEmulator pubSubEmulator;
 
-  private PubSubEmulator pubSubEmulator = new PubSubEmulator();
+  static {
+    try {
+      pubSubEmulator = new PubSubEmulator();
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
as this is a test class i'd be more inclined to do a fail call here rather than printing the stack trace